assets/css/_route_ladder.scss

```diff
@@ -1,25 +1,40 @@
 .m-route-ladder {
-  margin: 0 2rem;
-  width: 10rem;
+  margin: 0 0.5rem;
+  width: 12em;
 }
 
 .m-route-ladder__header {
-  background-color: whitesmoke;
-  border: 1px solid dimgray;
-  display: flex;
-  margin-bottom: 1rem;
-  padding: 0.2rem 0.5rem;
+  background-color: $sunflower;
+  border-radius: 4px;
+  font-size: 1.25rem;
+  font-weight: bolder;
+  margin-bottom: 2rem;
+  padding: 0.4rem 0;
+  position: relative;
   text-align: center;
 }
 
-.m-route-ladder__route-name {
-  flex-grow: 1;
-}
-
 .m-route-ladder__close {
   background: none;
   border: none;
+  color: $dark;
   cursor: pointer;
+  font-size: 1rem;
+  padding: 0;
+  position: absolute;
+  right: 0.5rem;
+  top: 0.5rem;
+}
+
+.m-route-ladder__close-icon {
+  display: inline-block;
+  height: 0.625rem;
+  width: 0.625rem;
+
+  svg {
+    stroke: $dark;
+    fill: $dark;
+  }
```

Comment:
This all seems pretty general. Could this be applied to all close icons, so we don't need to set the custom `className` for each one?

Reply:
Would you add something like `m-close-icon` to the svg itself, and set the css on that then?

Reply:
Or maybe we even just set this for all svg elements until we have one that differs?

Reply:
Made the svg bit generic.